tls/tls-full/client-accounting.conf

```diff
@@ -0,0 +1,15 @@
+[req]
+default_bits = 4096
```

Comment:
we are treating these as namespace-level certs right? is it worth putting "namespace" in the name of the conf file to make that clearer?